privval/signer_remote.go

```diff
@@ -7,20 +7,14 @@ import (
 
 	"github.com/pkg/errors"
 
-	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/go-amino"
```

Comment:
My IDE also complans about these aliases, and goimports does not add them on my machine. But we use this alias (and similar ones) everywhere. There was even a PR that consistently added them everywhere: https://github.com/tendermint/tendermint/pull/3219
which was the result of running `find . -name '*.go' | grep -v 'vendor/' | xargs -n 1 goimports -w` 
I'm pretty sure you'll be asked to add them back.

Reply:
:+1: 
maybe in the future the repos can be renamed to the aliases.
When you rename a repo in GitHub, it will automatically redirect the url is somebody tries to use the old one.